jupyterlab_latex/__init__.py

```diff
@@ -282,11 +288,14 @@ def load_jupyter_server_extension(nb_server_app):
     Called when the extension is loaded.
 
     Args:
-        nb_server_app (NotebookWebApplication): handle to the Notebook webserver instance.
+        nb_server_app (NotebookApp): handle to the Notebook webserver instance.
```

Comment:
For my own understanding, what is the difference between these two types?

Reply:
`NotebookApp` is a traitlets Application (actually a JupyterApplication), which allows traitlets to handle the command line arguments and configuration files, passing it in to the `NotebookWebApplication` via its settings.

`NotebookWebApplication` is a tornado Application which never directly uses traitlets.

Reply:
Cool, thanks. Is `nb_server_app` multiply inheriting from them, then?

Reply:
Ah, never mind, I see. The docstring was just wrong.